packages/compiler/src/render3/view/compiler.ts

```diff
@@ -64,9 +64,10 @@ function baseDirectiveFields(
   });
   definitionMap.set('factory', result.factory);
 
-  definitionMap.set('contentQueries', createContentQueriesFunction(meta, constantPool));
-
-  definitionMap.set('contentQueriesRefresh', createContentQueriesRefreshFunction(meta));
+  if (meta.queries.length) {
```

Comment:
```suggestion
  if (meta.queries.length > 0) {
```

I prefer to be explicit in conditionals, for 3 reasons:

1. Readability - it's obvious looking at the explicit comparison exactly what condition is being checked. Whereas looking at `if (foo)`, it's not really clear which values `foo` can take and which of them are being 

2. Correctness - it's easy to make mistakes and write expressions like `if (!template)`, which fails if `template` is an empty string.

3. Performance - JavaScript VMs emit far more code to support `if (x)` than `if (x > 0)`, because of the range of possible values. There's not a big difference, but it does exist.